File: src/ResumeTailor.Application/Accounts/Models/AccountRequest.cs


namespace ResumeTailor.Application.Accounts.Models;

public sealed record AccountRequest(
    string Auth0UserId,
    string Email,
    string DisplayName,
    string City,
    string State,
    string Country,
    IReadOnlyCollection<PersonalLinkRequest> PersonalLinks,
    IReadOnlyCollection<TitleRequest> Titles
    );

File: src/ResumeTailor.Client/Web/src/models/api/AccountRequest.ts
import type { PersonalLinkRequest } from "./PersonalLinkRequest";
import type { TitleRequest } from "./TitleRequest";

export type AccountRequest = {
  email: string;
  displayName: string;
  city: string;
  state: string;
  country: string;
  personalLinks: PersonalLinkRequest[];
  titles: TitleRequest[];
};
